dcos_installer/upgrade.py

```diff
@@ -71,6 +71,25 @@
     exit 1
 fi
 
+# Check if the node has node/cluster checks and run them
+if 3dt check; then
```

Comment:
We should be careful with this test, if `3dt` is broken then this'll cause us to silently skip checks. It'd be more reliable to test for the existence of the check config file.